### ISSUE TITLE
기능 추가 문의

### ISSUE THREAD

**Issue by Beolrok** [bullrock33/Paint#6]
지우개 기능을 사용할때 가시성의 문제가 있습니다
기존 그림을 그릴때는 연필 모양의 커서가 나와서 괜찮은데
지우개 기능을 사용시에는 커서가 아무거도 없어서 사용하는데 불편함을 겪을 것 같습니다

**Comment by Gyummybear** [comment]
맞습니다. 그릴 때는 연필 모양이 나와 직관적으로 그리는 기능을 작성하는게 보여, 좋지만 지우개는 아이콘 모양이 변하지 않아 불편함을 저도 겪고 있습니다.

**Comment by tjddnr57** [comment]
내일 회의 때 검토해봅시다
